AirCasting/SessionViews/DeleteSessionViewModel.swift

```diff
@@ -0,0 +1,28 @@
+// Created by Lunar on 16/07/2021.
+//
+
+import Foundation
+
+struct DeleteSessionOptionViewModel {
+    let id: Int
+    let title: String
+    var isSelected: Bool
+    let isEnabled: Bool
+    
+    mutating func toggleSelection() {
+        isSelected.toggle()
+    }
+    mutating func changeSelection(newSelected: Bool) {
+        isSelected = newSelected
+    }
+}
+
+protocol DeleteSessionViewModel: ObservableObject {
+    var isMicrophone: Bool { get }
```

Comment:
I don't really fancy this kind of shortcuts. We should aim at making this kind of viewmodels/views agnostic of session types. It's not super easy and not super important here and since we're on a hard deadline let's leave it like this and put a `#warning("Abstract microphone sessions away")` here and perhaps we can schedule a pair programming session in the future so I can show you how to do this

Reply:
see my second comment - this might be outdated if `isMicrophone` and other microphone-related are only here for mocking purposes